sql-plugin/src/main/301until330-all/scala/com/nvidia/spark/rapids/shims/v2/RapidsErrorUtils.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims.v2
+
+import ai.rapids.cudf.ColumnVector
+
+object RapidsErrorUtils {
+  def throwArrayIndexOutOfBoundsException(ordinalValue: Int, minNumElements: Int): ColumnVector = {
+    throw new ArrayIndexOutOfBoundsException(
+      s"Invalid index: $ordinalValue, minimum numElements in this ColumnVector: " +
```

Comment:
"minimum numElements" seems oddly worded -- was this from Spark code somewhere?  If not, this should just be something like, "$ordinalValue index beyond the $numElements elements".  Also this is hardcoded to assume the error is coming from a ColumnVector which is an assumption this utility code should not make.

Reply:
I just copied the old error from the `GpuElementAt`.